src/ankit/assignment_16/BankSwitch.java

```diff
@@ -0,0 +1,196 @@
+package ankit.assignment_16;
+
+import java.util.Scanner;
+
+public class BankSwitch {
+	int hBalance = 5000, iBalance = 6000, sBalance = 7000, bBalance = 8000;
+	int hDebit=0, hCredit=0, iDebit=0, iCredit=0, sDebit=0, sCredit=0, bDebit=0, bCredit;
+	static int tDebit, tCredit; 
+
+	void creditDebit(String str, int amt, String bank) {
+		switch(bank) {
+		case "HDFC" :
+			switch(str) {
+			case "Debit" :{
+				if(hBalance >= amt) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.